addOns/ascanrulesBeta/src/main/resources/org/zaproxy/zap/extension/ascanrulesBeta/resources/Messages.properties

```diff
@@ -161,7 +161,7 @@ ascanbeta.sessionfixation.alert.url.extrainfo = A likely session value has appea
 ascanbeta.sessionfixation.alert.url.extrainfo.loginpage = The url on which the issue was discovered was flagged as a logon page.
 ascanbeta.sessionfixation.desc = Session Fixation may be possible. If this issue occurs with a login URL (where the user authenticates themselves to the application), then the URL may be given by an attacker, along with a fixed session id, to a victim, in order to later assume the identity of the victim using the given session id. If the issue occurs with a non-login page, the URL and fixed session id may only be used by an attacker to track an unauthenticated user's actions. If the vulnerability occurs on a cookie field or a form field (POST parameter) rather than on a URL (GET) parameter, then some other vulnerability may also be required in order to set the cookie field on the victim's browser, to allow the vulnerability to be exploited.
 ascanbeta.sessionfixation.name = Session Fixation
-ascanbeta.sessionfixation.refs = https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A2-Broken_Authentication\nhttps://owasp.org/www-community/attacks/Session_fixation\nhttp://www.acros.si/papers/session_fixation.pdf\nhttp://www.technicalinfo.net/papers/WebBasedSessionManagement.html
+ascanbeta.sessionfixation.refs = https://owasp.org/www-project-top-ten/OWASP_Top_Ten_2017/Top_10-2017_A2-Broken_Authentication\nhttps://owasp.org/www-community/attacks/Session_fixation\nhttps://www.acros.si/papers/session_fixation.pdf\nhttps://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html
```

Comment:
The PDF is a 301 to https://acrossecurity.com/papers/session_fixation.pdf

Reply:
Does this mean I should revert the change for the pdf to keep the http? Or should I remove/find another reference?

Reply:
Updated to new reference

Reply:
You should use the final destination after the redirect

Reply:
Thank you, I did not know that was standard for reference links